src/confirm.rs

```diff
@@ -201,3 +201,30 @@ impl<'a> Confirm<'a> {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::test::without_ansi;
+    use indoc::indoc;
+
+    #[test]
+    fn test_render() {
+        let confirm = Confirm::new("Are you sure?")
+            .description("This will do a thing.")
+            .affirmative("Yes!")
+            .negative("No.");
+
+        assert_eq!(
+            indoc! {
+              " Are you sure? This will do a thing.
```

Comment:
is the description intended to be on the same line while adding an additional line break?

Reply:
no